.github/workflows/build-and-test-msi.yaml

```diff
@@ -80,15 +80,16 @@ jobs:
           role-session-name: windows-msi
           aws-region: ${{ secrets.WINDOWS_REGION }}
       - name: Remove Finch VM
-        timeout-minutes: 2
+        timeout-minutes: 5
         run: |
           $ErrorActionPreference = 'Ignore'
           taskkill /f /im wslservice.exe 2> nul || cmd /c "exit /b 0"
-          wsl --list --verbose
           sc query LxssManager | findstr "STATE" | findstr /C:"STOPPED" > nul && net start LxssManager
-          wsl --shutdown
-          wsl --unregister lima-finch
           wsl --list --verbose
+          wsl --list --verbose | findstr /C:"Running" > nul && wsl --shutdown
```

Comment:
i would say change the script to try to shutdown and wait for lets say 1 min, if it doesnt respond kill it from taskmanager**https://www.ninjaone.com/blog/how-to-kill-a-process-in-windows/#:~:text=Run%20the%20Windows%20kill%20process%20with%20PowerShell&text=Type%20%E2%80%9CGet%2DProcess%E2%80%9D%20to,Name%20%E2%80%9CProcessName%E2%80%9D%20%2DForce.**

that should fix it i suppose?

Reply:
https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/background-jobs?view=powershell-7.4

Reply:
Ack, added timeout and force kill logic in latest commit.